### PR TITLE
Transactions

### PR DESCRIPTION
**Pull request by nikolasburk**

### CODE REVIEW

File: docs/transactions.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      }
+    }
+  })
+  await tx.commit()
+})
+```
+
+In this case, the API provides a way to wrap a sequence of operations in a callback which gets executed as a transaction, therefore is guaranteed to either succeed or fail as a whole.
+
+If you'd like to see transactions supported in the future, [please join the discussion on GitHub](https://github.com/prisma/prisma/issues/4155).
```
Comment (janpio) [inline docs/transactions.md]: That link is super weird - shouldn't there be something in prisma2 or specs?

Comment (nikolasburk) [reply docs/transactions.md]: Thanks for catching this, yeah this must have been a mistake. I updated the link to an issue in the `prisma2` repo.

File: docs/transactions.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+```
+
+## Future transaction support in the Photon API
+
+Transactions are a commonly used feature in relational as well as non-relational databases and Photon might support more transaction mechanisms in the future. Specifically, the following two use cases will be supported:
+
+- Sending multiple operations in bulk.
+- Enabling longer-running transactions where operations can depend on each other.
+
+The first use use case of sending multiple operations in bulk could be implemented with an API similar to this:
```
Comment (deadcoder0904) [inline docs/transactions.md]: 👀`use use case` double `use`

Comment (janpio) [reply docs/transactions.md]:
```suggestion
The first use case of sending multiple operations in bulk could be implemented with an API similar to this:
```

**Pull request merged**